cmd/minishift/cmd/start.go

```diff
@@ -324,7 +324,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 	if !isRestart {
 		fmt.Println("-- Minishift VM will be configured with ...")
-		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   Memory:   ", units.BytesSize(float64(machineConfig.Memory*units.MiB)))
```

Comment:
Using `HumanSize` is not correct for this. Also, as an example, libvirt/kvm internally use KiB for sotring the value (and gets displayed in virt-manager as GiB). I do not see the problem with the conversion so much. This is pretty much like 1000cc is 1L, or 1 HP is 745.69 Watt or 45.69 Nm/s. As long as a visual indication is given.

However, storing the value as-is might be possible... but, this is only shown when creating the VM. The valus is stored inside the `config.json` as:

```json
{
    "ConfigVersion": 3,
    "Driver": {
        "Memory": 4096,
        "DiskSize": 20000,
```

Showing all values as MiB seems like a workaround. Perhaps it is me, but I do not really talk about megabytes anymore, but rather gigabytes. Even for a cloud instance, it is often 0.5G instead of 512M. What I think is the most troubling, is showing a value smaller than 1G as 0. Although this could solve that possible problem, all values will be shown as a 4 digit number at least.

But the question here is, do we really need to be fine-grained about the VM creation specifications? It seems people in the field do not care much about precision:

```patch
-    $ minishift start --memory=5000 --cpus=5 --disk-size=50g
+    $ minishift start --memory=5120 --cpus=5 --disk-size=50g
```

Ref: https://github.com/minishift/minishift-addons/issues/20

I guess they want to create an instance with 5G and would not bother much when they see 5GiB as the unit displayed.